internal/config/golang.go

```diff
@@ -0,0 +1,11 @@
+package config
+
+import "github.com/spf13/viper"
+
+type golang struct {
+	SearchLocalModCacheLicenses bool `json:"search-local-mod-cache-licenses" yaml:"search-local-mod-cache-licenses" mapstructure:"search-local-mod-cache-licenses"`
```

Comment:
"Local" could be ambiguous (local to where? the container image? the host?). What about "Host"?

```
search-host-mod-cache-licenses
```

(not trying to nit pick to death, but since it's on the user surface area this is easy to change now, hard to change later)

Reply:
If you're scanning a directory... is that still the "host" (not like host vs container is more obvious)?

Reply:
I guess it could be ambiguous any way you slice it:
- on host, dir scanning on the host: "host" makes sense
- on host, scanning an image: "host" makes sense
- in container, dir scanning in the container: "host" would be wrong in an absolute sense
- in container, scanning an image : "host" would be wrong in an absolute sense

I take back my comment -- "local" is good enough (still suffers from some ambiguity, but better than than being wrong in an absolute sense).